docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md

```diff
@@ -6,7 +6,7 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 <Tabs>
-  <TabItem value="Rancher v2.6.7">
+  <TabItem value="Rancher v2.7.0+">
```

Comment:
The flow here with this change is kind of odd. There's now two tabs - one for v2.6.0-v2.6.6 and one for v2.7.0+. It kind of makes it seem like azureAD is not supported on 2.6 versions after 2.6.6. Can you elaborate on why this change was made?

Reply:
Mostly because these docs are v2.7 docs, so any v2.6 references are not relevant for it.

Reply:
From my side, I see these docs with a drop-down of 2.6 at the top and there's another tab for `Rancher v2.6.0 - v2.6.6`. Given what you have said here, I would expect to only see stuff for 2.7 - is this intentional?

Reply:
This is known - either the docs team have not yet updated it, or they have, and I just haven't rebased against a more recent version of the branch.

Reply:
I think that this makes for a confusing user experience. If we can fix this, I would recommend doing so. If these docs are also intended for 2.7.0+, I think that it makes sense to not have a tab for 2.7.0+ - it's already on the 2.7 page, so users know that it is for 2.7.

Reply:
@btat, what do you think? I am also not sure how to remove a tab item without breaking the page...